Runtime/Scripts/EdgeEventsManager.cs

```diff
@@ -553,7 +573,6 @@ void PropagateError(FindCloudletEventTrigger trigger, string error_msg)
       findCloudletEvent.trigger = trigger;
       Debug.LogError(error_msg);
       EdgeEventsStatus eventStatus = new EdgeEventsStatus(Status.error, error_msg);
-      StopEdgeEvents();
```

Comment:
Only this change here involving EdgeEvents logic @brucearmstrong 
Basically don't stop edge events if there is an error